src/frontend/src/pages/WorkPackageTemplateForm/WorkPackageTemplateFormView.tsx

```diff
@@ -0,0 +1,169 @@
+  } = useFieldArray({ control, name: 'descriptionBullets' });
+
+  const onSubmit = async (data: WorkPackageTemplateFormViewPayload) => {
+    const { workPackageName, templateName, templateNotes, duration, blockedBy, stage, descriptionBullets } = data;
+
+    const blockedByWbsNums = blockedBy.map((blocker) => blocker.id);
+
+    try {
+      const payload: WorkPackageTemplateApiInputs = {
+        templateName,
+        templateNotes,
+        duration,
+        stage: stage as WorkPackageStage,
+        blockedBy: blockedByWbsNums,
+        descriptionBullets,
+        workPackageName
+      };
+
+      await workPackageTemplateMutateAsync(payload);
+      toast.success('Work Package Template submitted successfully');
+      history.push(routes.ADMIN_TOOLS + '/project-configuration/work-package-templates');
+    } catch (error) {
+      toast.error('Error submitting work package template');
+      console.error('Error submitting work package template:', error);
+    }
+  };
+
+  return (
+    <form
+      id="work-package-template-edit-form"
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit((data) => {
+          onSubmit(data);
+        })(e);
+      }}
+      onKeyPress={(e) => {
+        e.key === 'Enter' && e.preventDefault();
+      }}
+    >
+      <PageLayout
+        stickyHeader
+        title={pageTitle}
+        headerRight={
+          <Box display="inline-flex" alignItems="center" justifyContent={'end'}>
+            <Box>
+              <NERButton variant="contained" onClick={exitActiveMode} sx={{ mx: 1 }}>
+                Cancel
+              </NERButton>
+              <NERSuccessButton variant="contained" type="submit" sx={{ mx: 1 }}>
+                Submit
+              </NERSuccessButton>
+            </Box>
+          </Box>
+        }
+      >
+        <Stack spacing={2}>
+          <Box my={2}>
+            <WorkPackageTemplateFormDetails control={control} errors={errors} />
+          </Box>
+          <Typography variant="h5">Blocked By</Typography>
+          <FormControl fullWidth>
+            <Controller
+              name="blockedBy"
+              control={control}
+              render={({ field: { onChange, value: formValue } }) => (
+                <Autocomplete
+                  isOptionEqualToValue={(option, value) => option.id === value.id}
+                  filterSelectedOptions
+                  multiple
+                  options={blockedByOptions}
+                  getOptionLabel={(option) => option.label}
+                  onChange={(_, value) => onChange(value)}
+                  value={formValue}
+                  renderInput={(params) => (
+                    <TextField {...params} variant="standard" placeholder="Select Blockers" error={!!errors.blockedBy} />
+                  )}
+                />
+              )}
+            />
+          </FormControl>
+          <Box>
+            <DescriptionBulletsEditView
```

Comment:
Could this go in the same file as the other fields or do you think it won't fit?